app/src/userpages/components/ProfilePage/ProfileSettings/index.jsx

```diff
@@ -77,41 +67,39 @@ export class ProfileSettings extends Component<Props> {
         }
         return (
             <Fragment>
-                <Form onSubmit={this.onSubmit}>
-                    <Avatar
-                        className={styles.avatar}
-                        editable
-                        // $FlowFixMe
-                        user={user}
-                        onImageChange={this.onImageChange}
+                <Avatar
+                    className={styles.avatar}
+                    editable
+                    // $FlowFixMe
+                    user={user}
+                    onImageChange={this.onImageChange}
+                />
+                <div className={styles.fullname}>
+                    <TextInput
+                        label="Your name"
+                        name="name"
+                        value={user.name || ''}
+                        onChange={this.onNameChange}
+                        required
+                        preserveLabelSpace
                     />
-                    <div className={styles.fullname}>
-                        <TextInput
-                            label="Your name"
-                            name="name"
-                            value={user.name || ''}
-                            onChange={this.onNameChange}
-                            required
-                            preserveLabelSpace
-                        />
-                    </div>
-                    <div className={styles.email}>
-                        <TextInput label="Email" value={user.username || ''} readOnly />
-                    </div>
-                    <div className={styles.password}>
-                        <ChangePassword.Button />
-                    </div>
-                    <div className={styles.timezone}>
-                        <SelectInput
-                            label="Timezone"
-                            name="name"
-                            options={options}
-                            value={options.find(({ value }) => user.timezone === value)}
-                            onChange={this.onTimezoneChange}
-                            required
-                        />
-                    </div>
-                </Form>
+                </div>
+                <div className={styles.email}>
+                    <TextInput label="Email" value={user.username || ''} readOnly />
+                </div>
+                <div className={styles.password}>
+                    <ChangePassword.Button />
+                </div>
+                <div className={styles.timezone}>
```

Comment:
I think Timezones are on their way out, is there any point in keeping this SelectInput?

Reply:
I have no idea, it's in the design 🤷‍♂️